src/init-agent-script.ts

```
@@ -363,1219 +144,1303 @@
-              myGroupMsgSenderIdPtr = Memory.alloc(groupMsgSenderIdLen)
-              Memory.copy(myGroupMsgSenderIdPtr, groupMsgSenderIdPtr, groupMsgSenderIdLen)
+}
 
-            }
+/*
+发送位置消息
+*/
+async function messageSendLocation(
+  conversationId: string,
+  locationPayload: any,
+): Promise<void | string> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)